projects.yaml

```diff
@@ -277,4 +277,8 @@ projects:
     sam3u2c_ncs36510rf_if:
         - *module_if
         - *module_hic_sam3u2c
-        - records/board/ncs36510rf.yaml        
+        - records/board/ncs36510rf.yaml
+    lpc11u35_mtconnect04s:
```

Comment:
Interface firmware needs to end with `_if`, so this should be `lpc11u35_mtconnect04s_if`. Also, you'll want to add your target to `SUPPORTED_CONFIGURATIONS` and `PROJECT_RELEASE_INFO` in test/info.py so it is recognized by testing and created as part of the release process.